meinberlin/react/livequestions/LikeCard.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import django from 'django'
+import { Pill } from '../contrib/Pill'
+import { classNames } from '../contrib/helpers'
+
+export default function LikeCard ({
+  title,
+  category,
+  likes,
+  onClickLike
+}) {
+  const likesTag = likes.session_like
+    ? django.gettext('Liked')
+    : django.gettext('Likes')
+  const addLikeTag = django.gettext('add like')
+  const undoLikeTag = django.gettext('undo like')
+
+  return (
+    <article className="modul-card card">
+      <header className="card__header">
+        <h3 className="title">{title}</h3>
+        {category && (
+          <div className="card__status status">
+            <Pill pillClass="card__pill pill pill--label">{category}</Pill>
+          </div>
+        )}
+      </header>
+      {onClickLike
+        ? (
+          <button
+            type="button"
+            className={classNames(
+              'rating-button',
+              likes.session_like && 'rating-button--active'
+            )}
+            onClick={() => onClickLike()}
```

Comment:
```suggestion
            onClick={onClickLike}
```
unless there is an issue if the function receives the event parameter.